src/frontend/barnetilsyn/tekster/aktivitet.ts

```diff
@@ -197,7 +197,7 @@ export const aktivitetTekster: AktivitetInnhold = {
         header: { nb: 'Søke lengre tilbake enn 3 måneder?' },
         innhold: {
             nb: [
-                'Hvis du skal søke i forbindelse med en aktivitet som ble avsluttet for over 3 måneder siden, må du fylle ut ',
+                'Du må sende inn søknad på papir hvis du skal søke om en aktivitet som ble avsluttet for mer enn 3 måneder siden. ',
```

Comment:
Ser ut som papirsøknad lenken ble litt rar på slutten her. Tror dette vil funke:

```suggestion
             nb: [  
                'Du må sende inn ',
                {
                    tekst: 'søknad på papir',
                    url: 'https://tjenester.nav.no/soknadtilleggsstonader/app/start',
                    variant: 'neutral',
                },
                ' hvis du skal søke om en aktivitet som ble avsluttet for mer enn 3 måneder siden.',
            ],
```